main/models/db.js

```diff
@@ -0,0 +1,26 @@
+const { Pool } = require('pg');
+// require('dotenv').config();
+
+// For saving the database URI in the env file instead:
+// const pool = new Pool({ connectionString: process.env.DB_URI });
+
+const pool = new Pool({
+  connectionString:
+    'postgres://gmasbmxl:R44JPMUT3LjuZqZ2Hcmn2A65P4k2Lv-B@berry.db.elephantsql.com/gmasbmxl',
```

Comment:
can we put this in the .env?

Reply:
updated

Reply:
@echochs1 as discussed, removed injection of pool to request object in expressServer.js